From 31a2c5555f9ef32f35d7d5ce1fd09a010ba5f5c6 Mon Sep 17 00:00:00 2001
From: Joshua Watt <JPEWhacker@gmail.com>
Date: Mon, 17 Jun 2019 10:47:15 -0500
Subject: [PATCH 1/2] enc2xs: Add environment variable to suppress comments

Comment generation in enc2xs can now be suppressed by setting the
ENC2XS_NO_COMMENTS environment variable. This allows enc2xs to produce
reproducible output by omitting the name of the generating program.

Signed-off-by: Joshua Watt <JPEWhacker@gmail.com>
Upstream-Status: Accepted [https://github.com/dankogai/p5-encode/pull/145]
---
 cpan/Encode/bin/enc2xs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cpan/Encode/bin/enc2xs b/cpan/Encode/bin/enc2xs
index 619b64b757..bfce9ee735 100644
--- a/cpan/Encode/bin/enc2xs
+++ b/cpan/Encode/bin/enc2xs
@@ -144,6 +144,7 @@ getopts('CM:SQqOo:f:n:v',\%opt);
 $opt{M} and make_makefile_pl($opt{M}, @ARGV);
 $opt{C} and make_configlocal_pm($opt{C}, @ARGV);
 $opt{v} ||= $ENV{ENC2XS_VERBOSE};
+$opt{q} ||= $ENV{ENC2XS_NO_COMMENTS};
 
 sub verbose {
     print STDERR @_ if $opt{v};
-- 
2.21.0

